src/components/Popover/Popover.tsx

```diff
@@ -101,9 +105,9 @@ export const Popover: FC<PopoverProps> = ({
   offsetFromTarget = 12,
   onClickOutside = undefined,
   placement = 'right',
-  portalTarget = document.body,
-  trapFocus = false,
   withPortal = false,
+  portalTarget = withPortal ? document.body : undefined,
```

Comment:
This has potential to still cause issues in SSR contexts if withPortal is enabled, but I think Portals cause problems anyway so that's a problem for another day!